docs/usage/upgrade-best-practices.md

```diff
@@ -0,0 +1,278 @@
+# Upgrade best practices
+
+This page explains what we (the Renovate maintainers) recommend you do to update your dependencies.
+
+We'll cover starting a new project, updating a year-old project, and updating a project with five year old dependencies.
+We explain why you should update often, and how to nudge your team to update their dependencies.
+
+## General recommendations
+
+In general, you should:
+
+- Run Renovate on _every_ repository
+- Use the `config:best-practices` preset instead of `config:recommended`
+- Use the Dependency Dashboard issue (it's on by default)
+- Update your dependencies often
+- Read the changelogs for the updates
+- Update to new `major` versions in good time
+- Talk with your team about the update strategy
+
+If Renovate is too noisy for you, read the [noise reduction docs](./noise-reduction.md).
+
+## Use the `config:best-practices` preset
+
+The `config:recommended` preset is the recommended configuration for most Renovate users.
+Renovate also has a `config:best-practices` preset that includes our upgrade best practices.
+
+You should extend from the `config:best-practices` preset:
+
+```json
+{
+  "extends": ["config:best-practices"]
+}
+```
+
+If you're using `config:recommended` now, replace it with `config:best-practices`:
+
+```diff
+- "extends": ["config:recommended"]
++ "extends": ["config:best-practices"]
+```
+
+### What's in the `config:best-practices preset?
+
+The [`config:best-practices` preset](https://docs.renovatebot.com/presets-config/#configbest-practices) has this configuration:
+
+```json
+{
+  "configMigration": true,
+  "extends": [
+    "config:recommended",
+    "docker:pinDigests",
+    "helpers:pinGitHubActionDigests",
+    ":pinDevDependencies"
+  ]
+}
+```
+
+The next sections explain each part of the preset.
+
+#### Config migration
+
+Renovate creates a config migration PR to replace old config option names with their new replacements.
+This way your configuration file and the Renovate docs always use the same terms.
+
+You'll get config migration PRs no matter _how_ you run Renovate: self-hosting or the Mend Renovate app.
+
+#### Extends `config:recommended`
+
+The `config:recommended` preset is a good base to start from.
+That's why we extend from it.
+
+#### Extends `docker:pinDigests`
+
+The [Renovate docs, Docker Digest pinning](./docker.md#digest-pinning) section explains _why_ you should pin your Docker containers to an exact digest.
+
+#### Extends `helpers:pinGitHubActionDigests`
```

Comment:
@HonkingGoose I just read a really nice real-life example for why we want this:
https://www.paloaltonetworks.com/blog/prisma-cloud/github-actions-worm-dependencies/

Reply:
Thank you for the article! This is the basic attack:

> We can use the secrets exfiltrated in the flow to infect the repository with malicious code. Overwriting a commit while keeping its hash the same isn’t possible, so we can’t abuse a commit hash reference.

And they recommend:

> 4. Pin actions using a commit hash to reduce the risk of using a maliciously modified action.

Reply:
Yep, basic, which means it's really easy, and easy to protect against too. :)

Reply:
We could link to the article, or keep things the same.

In favor of the article would be that it explains why pinning to the commit hash is good, and it helps strengthen our "pin all actions"-stance.

Against the article is that it may be too much information.

I'll let the maintainer decide. 😉

Reply:
I'm fine with linking to the blog if the user wants some more detailed information. 

Something like: 
```
For an in depth explanation why you should pin your Github Actions see: 
```
or something similar.